wiki/build_system.md

```
@@ -0,0 +1,325 @@
+make   # new_enemy.c is compiled automatically
+```
+
+See [Developer Guide](developer_guide) for the full new-entity workflow.
+
+---
+
+## Output Structure
+
+After a successful build:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)